packages/iris-grid/src/ColumnStatistics.jsx

```diff
@@ -169,6 +169,12 @@ class ColumnStatistics extends Component {
         {description && (
           <div className="column-statistics-description">{description}</div>
         )}
+        {!model.isColumnMovable(column.name) && (
```

Comment:
As mentioned in another comment, `isColumnMovable` should take a column index instead of a name. So you'll need to make a method here that you can call, and essentially get the index by calling `model.columns.findIndex(col => col.name === column.name)`, then call `isColumnMovable` with that index.

IrisGridModel has the concept of columns with names, but GridModel does not. So need to respect the GridModel interface.